src/index.html

```diff
@@ -7,12 +7,61 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="card__img"
+      />
+
+      <p class="card__info">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="card__productCode">Product code: 195434</p>
+
+      <div class="stars">
+        <div class="stars__container">
```

Comment:
Add the modifier class
```suggestion
        <div class="stars stars--4">
```